R/per_capita_household_size.R

```diff
@@ -0,0 +1,42 @@
+#' @title Get per capita household size with household size distribution
+#' @description Returns the per capita household size for a location given its household size distribution. 
+#' See [get_household_size_distribution()] function for retrieving household size distributions for a given place.
+#' @param household_data data set with information on the household size distribution of specific state or LGA. 
+#' @param household_size bare variable name of the column depicting the household size. Default is 'household_size' from [get_household_size_distribution()].
+#' @param n_people bare variable name of the column depicting the total number of people belonging to the respective household size. Default is 'n_people' from [get_household_size_distribution()]. 
+#' @return Numeric of length 1 - the per capita household size for a given state
+#'     or LGA.
+#' @author Nick Golding
+#' @export
+#' @examples 
+#' sample_data <- get_household_size_distribution(lga = "Fairfield (C)")
```

Comment:
Can you print out `sample_data` in the example in the next line, so users can see the columns on the online docs?